.github/workflows/main.yml

```diff
@@ -39,6 +39,10 @@ jobs:
             msys-env: mingw-w64-ucrt-x86_64
 
     steps:
+      - name: Set up Homebrew
```

Comment:
I'd rather backport the libsndfile package from ubuntu 23.04 than mix in another package manager.

Reply:
I am probably doing this wrong, but it seems that `libsndfile` is not in the jammy-backports archive? So should we do it ourselves?